src/main/java/server/UserService.java

```diff
@@ -0,0 +1,21 @@
+package server;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+class UserService {
+    private final HashSet<String> users = new HashSet<String>();
+
+    UserService() {
+        this.users.add("Anders");
+        this.users.add("Lukas");
+        this.users.add("Nicklas");
+        this.users.add("Simon");
+    }
+
+
+    public boolean usernameExists(String name) {

```

Comment:
This helps us abstract away access to the set itself. We could modify this method to add synchronization primitives, or reach out to a database or any other number of things without the caller having to be aware of it.